azure_monitor/src/azure_monitor/export/metrics/__init__.py

```diff
@@ -99,3 +112,43 @@ def _metric_to_envelope(
         data = protocol.MetricData(metrics=[data_point], properties=properties)
         envelope.data = protocol.Data(base_data=data, base_type="MetricData")
         return envelope
+
+
+def standard_metrics_processor(envelope):
+    data = envelope.data.base_data
+    if data.metrics:
+        properties = {}
+        point = data.metrics[0]
+        if point.name == "http.client.duration":
+            point.name = "Dependency duration"
+            point.kind = protocol.DataPointType.AGGREGATION.value
+            properties["_MS.MetricId"] = "dependencies/duration"
+            properties["_MS.IsAutocollected"] = "True"
+            properties["cloud/roleInstance"] = utils.azure_monitor_context.get(
+                "ai.cloud.roleInstance"
+            )
+            properties["cloud/roleName"] = utils.azure_monitor_context.get(
+                "ai.cloud.role"
+            )
+            properties["Dependency.Success"] = "False"
+            if data.properties.get("http.status_code"):
+                try:
+                    code = int(data.properties.get("http.status_code"))
+                    if 200 <= code < 400:
+                        properties["Dependency.Success"] = "True"
+                except ValueError:
+                    pass
+            # TODO: Check other properties if url doesn't exist
```

Comment:
What kind of other properties?

Reply:
https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/metrics/semantic_conventions/http-metrics.md#label-alternatives